### ISSUE TITLE
testground healthcheck (--repair) command

### ISSUE THREAD

**Issue by nonsense** [ipfs/testground#426]
See design notes in epic.

---

We might want to create Testground a command, such as `testground validate-cluster` or something like this, which does a set of checks that the cluster is setup correctly:
- confirms that `sidecar` is running on every host.
- confirms that `redis` is running
- confirms that routes on hosts are setup correctly

`kops` has a similar tool:

```
➜  testground git:(less-steps) kops validate cluster simple.k8s.local
Validating cluster simple.k8s.local

INSTANCE GROUPS
NAME                    ROLE    MACHINETYPE     MIN     MAX     SUBNETS
master-eu-central-1a    Master  c5.2xlarge      1       1       eu-central-1a
nodes                   Node    c5.2xlarge      14      14      eu-central-1a

NODE STATUS
NAME                                            ROLE    READY
ip-172-20-34-37.eu-central-1.compute.internal   node    True
ip-172-20-39-154.eu-central-1.compute.internal  master  True
ip-172-20-39-244.eu-central-1.compute.internal  node    True
ip-172-20-45-108.eu-central-1.compute.internal  node    True
ip-172-20-48-175.eu-central-1.compute.internal  node    True
ip-172-20-52-192.eu-central-1.compute.internal  node    True
ip-172-20-53-4.eu-central-1.compute.internal    node    True
ip-172-20-58-197.eu-central-1.compute.internal  node    True
ip-172-20-58-89.eu-central-1.compute.internal   node    True
ip-172-20-59-73.eu-central-1.compute.internal   node    True
ip-172-20-60-196.eu-central-1.compute.internal  node    True
ip-172-20-61-105.eu-central-1.compute.internal  node    True
ip-172-20-61-144.eu-central-1.compute.internal  node    True
ip-172-20-61-26.eu-central-1.compute.internal   node    True
ip-172-20-63-3.eu-central-1.compute.internal    node    True

Your cluster simple.k8s.local is ready
```

---

I think this is useful specially while we are in the `self-serviced` mode, where folks are running individual k8s clusters, and don't necessarily know all the details around the inner-workings of the setup - even for the 3 things listed above, that'd be a good checklist to confirm that everything is setup and working as expected.

**Comment by raulk** [comment]
@coryschwartz since you're picking this up, please read through the design notes in the epic ;-)

**Comment by raulk** [comment]
@coryschwartz has focused on the `terminate` command. I'll be picking this one up.

**Comment by raulk** [comment]
This was implemented in #557.
